WalletWasabi.Fluent/Helpers/NotificationHelpers.cs

```diff
@@ -68,23 +71,25 @@ private static bool TryGetNotificationInputs(ProcessedResult result, [NotNullWhe
 				Money incoming = receivedSum - spentSum;
 				Money receiveSpentDiff = incoming.Abs();
 				string amountString = receiveSpentDiff.ToFormattedString();
+				string fiatString = receiveSpentDiff.ToUsd(fiatExchangeRate).ToUsdAmount();
```

Comment:
I was wondering if `ToUsdAproxBetweenParens()` instead of `ToUsdAmount()` would make more sense? Because of the `≈` sing, and later in strings, you don't have to deal with USD text and parenthesis.